.gitlab/pkg_metrics.yml

```diff
@@ -205,6 +235,50 @@ send_pkg_size-a7:
     # Make the job fail if at least one package is above threshold
     - if [ "${failures}" -ne "0" ]; then false; fi
 
+.check_arm64_pkg_size:
```

Comment:
This looks like a lot of duplicated code from `.check_pkg_size`. I think it should be possible to modify `.check_pkg_size` to work with an `ARCH` variable and then specify that variable on the `check_pkg_size*` jobs.

For the existing `check_pkg_size*` jobs, let's rename them to reflect that their for amd64.

Reply:
Should be good now I'll wait after the pipeline finishes to start an all_build pipeline, this way I'm sure nothing is broken